src/style.css

```diff
@@ -0,0 +1,125 @@
+.body {
+  margin: 0;
+}
+
+.product {
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__wrapimg {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.product__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0;
+}
+
+.product__button {
+  text-decoration: none;
+  width: 100%;
+  display: block;
+  padding: 12px 0;
+  background: #00acdc;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  border: 1px solid #00acdc;
+}
+
+.product__button:hover {
+  background: #fff;
+  color: #00acdc;
+}
+
+.review {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  margin: 16px 0 24px;
+}
+
+.review__star {
+  display: flex;
+}
+
+.review__stars {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) no-repeat;
+  margin-right: 4px;
+  background-position: center center;
+}
+
+.review__stars:last-child {
+  margin-right: 0;
+}
+
+.review__star--stars5 .review__stars,
+.review__star--stars1 .review__stars:first-child,
+.review__star--stars2 .review__stars:not(:nth-child(n+3)),
+.review__star--stars3 .review__stars:not(:nth-child(n+4)),
+.review__star--stars4 .review__stars:not(:nth-child(n+5)) {
+  background: url(images/star-active.svg) no-repeat;
+  background-position: center center;
+}
+
+.review__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  margin: 0;
+  display: flex;
+  align-self: center;
+}
+
+.product__price {
+  font-family: "Roboto", sans-serif;
```

Comment:
you can remove it form card elements - it font will be inherited
get rid of duplicates